src/quote/Quote.ts

```diff
@@ -108,3 +108,81 @@ export function createAgreedQuote(
     claimerSignature: signature,
   }
 }
+
+export function compressCost(quote: IQuote): any {
+  return Object.values(quote.cost)
+}
+
+export function decompressCost(quoteArray: IQuote['cost']): any {
+  const quote = quoteArray
+  return { gross: quote[0], net: quote[1], tax: quote[2] }
```

Comment:
Question:
Why are not all your objects prettified/eslinted in the same way? (see function right below)

Reply:
I guess it fits all on one line.

Reply:
#Unsure